spinnaker_camera_driver/cmake/download_spinnaker

```diff
@@ -87,32 +183,16 @@ if not os.path.exists(os.path.join(os.getcwd(), "usr/lib/")):
     logging.info("Unpacking debs.")
     debs = glob.glob(os.path.join(os.getcwd(), folder_name, "libspinvideoencoder-*.deb"))
```

Comment:
This deb doesn't seem to be included any more.  If the package works without it this line should be removed. It may have become libspinvideo.

Reply:
Removed.